apps/protocol-frontend-e2e/src/integration/06_create2ndContribution.cy.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+  before(() => {
+    cy.fixture('contributions.json').then(contributions => {
+      this.contributions = contributions;
+    });
+  
+    cy.fixture('testaccounts.json').then(accounts => {
+      this.accounts = accounts;
+      cy.login(this.accounts[0].address, this.accounts[0].privateKey);
+    });
+  
+    cy.get('[data-cy="myContributions-btn"]', { timeout: 15000 })
+      .should('be.visible')
+      .click({ force: true });
+
+    cy.get('[data-testid="floatingreportbtn-testid"]', {
+        timeout: 10000,
+      }).should('be.visible')
+        .click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(5000)
+  });
+
+describe("Create Second Contribution", () => {
+  it('Report your Second Contribution', () => {
+    const contribution = this.contributions[2];
+
+    cy.get('input[data-testid="reportForm-name"]', {timeout:20000})  
+      .should('be.visible')
+      .type(contribution.name)
+      .should('have.value', contribution.name);
+
+    cy.get('.css-ujecln-Input2')
```

Comment:
Is it possible to fetch this element by the id?

Reply:
I tried passing in test-Id in props for both creatable and select just like you did for Input. But the test-id didn't show up in the DOM :(